Makefile

```diff
@@ -1,3 +1,6 @@
+init:
+	go get -u github.com/jstemmer/go-junit-report
```

Comment:
Can we vendor it to use the known version?

Reply:
It has no releases. Only the master branch. :( or do you mean something else?

Reply:
Nothing prevents us from vendoring a specific commit with Gopkg.lock. It is still better than using current `master` at the time of running tests (as opposed to the time of `dep ensure -update`).

Reply:
I mean we use binary and we don't use it in any source code, do it will be deleted from Gopkg.toml after first `dep ensure`.

In short, I don't know how external binary should be integrated via dep...

Reply:
https://github.com/percona/pmm-agent/blob/ae64344161480bfc3ecc7c007bfb860ee3f0c86b/Gopkg.toml#L5-L15

Reply:
@AlekSi PTAL is it Ok? I've made changes.